### ISSUE TITLE
Colons in Nakama Leaderboard ID

### ISSUE THREAD

**Issue by amerryma** [heroiclabs/nakama#306]
## Description
Using the leaderboard feature for Nakama (2.4), I'm having an issue using a colon `:` in a leaderboard_id. Right now I'm getting a 500 error using the rest api to fetch the data. Example: `GET /v2/leaderboard/some:leaderboard`. I am able to create the leaderboard and add leaderboard records, but not fetch anything.

## Steps to Reproduce
1. Create a leaderboard with Lua `pcall(nk.leaderboard_create, "test:leaderboard", false, "desc", "incr", cron, metadata)`
1. Create a leaderboard record with Lua: `pcall(nk.leaderboard_record_write, "test:leaderboard", ownerId, userId, pointValue)`
1. Use Rest API to Fetch Data: 
   ```
   curl -H "Authorization: Bearer <bearer>" "localhost/v2/leaderboard/test%3Aleaderboard?limit=10"
   ```
   or
   ```
   curl -H "Authorization: Bearer <bearer>" "localhost/v2/leaderboard/test:leaderboard?limit=10"
   ```

## Expected Result
Returns the leaderboard records.

## Actual Result
Returns a 404 Not Found (no logs).

## Context
- [x] Rest API & Lua RPC

## Your Environment
- Nakama: 2.4.0+a5dd5427
- Cockroach: v2.0.1
- Environment name and version: ?
- Operating System and version: https://hub.docker.com/r/heroiclabs/nakama/tags @2.4.0

**Comment by amerryma** [comment]
Just a follow up. This may be related to this: https://github.com/grpc-ecosystem/grpc-gateway/issues/224

**Comment by zyro** [comment]
@amerryma I can reproduce it too. It looks like it's definitely related to https://github.com/grpc-ecosystem/grpc-gateway/issues/224 as you say.

I'll look into workarounds but your best bet for now is to avoid `:` characters in the leaderboard ID.

**Comment by zyro** [comment]
https://github.com/grpc-ecosystem/grpc-gateway/issues/224 appears to be resolved. This issue should be fixed through the dependency update in c97ad153cf216cd9586ad668219adc125ed79451, and will be part of the next release.
